app/helpers/logs_helper.rb

```diff
@@ -63,4 +64,16 @@ def logs_and_soft_validations_warning(bulk_upload)
   def bulk_upload_error_summary(bulk_upload)
     "You have tried to upload #{bulk_upload.total_logs_count ? pluralize(bulk_upload.total_logs_count, 'log') : 'logs'}. There are errors in #{pluralize(bulk_upload.logs_with_errors_count, 'log')}, and #{pluralize(bulk_upload.bulk_upload_errors.count, 'error')} in total."
   end
+
+  def deleted_errors_warning_text(bulk_upload)
+    unique_answers_to_be_cleared_count = unique_answers_to_be_cleared(bulk_upload).count
+    this_or_these = unique_answers_to_be_cleared_count == 1 ? "this" : "these"
+    it_is_or_they_are = unique_answers_to_be_cleared_count == 1 ? "it is" : "they are"
+
+    "#{pluralize(unique_answers_to_be_cleared_count, 'answer')} will be deleted because #{it_is_or_they_are} invalid. You will have to answer #{this_or_these} #{'question'.pluralize(unique_answers_to_be_cleared_count)} again on the site."
+  end
```

Comment:
Is there a situation where `unique_answers_to_be_cleared_count` is 0?

Reply:
yep, there's an edge case where there are errors but they're all "not_answered" errors so won't really be cleared because they're already blank:
 
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/806080b9-96f1-413e-80d2-85188786ee22)

Reply:
I think not all the copy makes sense though, I'll send this sc to olivia and susanna and see what they think

Reply:
because the two errors are the :not_answered type and so not included in the count, but still errors - olivia and I agreed it would be misleading not to show them as the user might think everything was fine and then be surprised that they had missing data in the service (when they had just failed to fill it in at all)

Reply:
we now don't display the link to this page in this case 👍